### ISSUE TITLE
Visual appearance of testnet site accounts

### ISSUE THREAD

**Issue by QustomQure** [iron-fish/ironfish#970]
### Description

Visual representation of community contributions is off the block when points number are four digits and more(too huge for rectangle block). Same should be for promotions rectangle if points digits will become too wide
![2022-02-09_01-06-22](https://user-images.githubusercontent.com/43998465/153084534-cb034e1c-3a25-4c29-9a9b-ce628823d89e.png)
.

**Comment by wd021** [comment]
should transfer issue to [website-testnet](https://github.com/iron-fish/website-testnet) repo

**Comment by dguenther** [comment]
Yep, transferring to [website-testnet](https://github.com/iron-fish/website-testnet).
